velox/substrait/SubstraitToVeloxPlan.h

```diff
@@ -25,6 +25,25 @@ namespace facebook::velox::substrait {
 /// This class is used to convert the Substrait plan into Velox plan.
 class SubstraitVeloxPlanConverter {
  public:
+  struct SplitInfo {
+    SplitInfo() {}
+
+    /// The Partition index.
+    u_int32_t partitionIndex_;
```

Comment:
naming: members of the struct should not have underscore at the end, i.e. partitionIndex